frontend/src/components/shared/mainInfo/MainInfo.tsx

```diff
@@ -23,9 +23,14 @@ const MainInfo: React.FC = () => {
         <InputWithFormik label="Name" name="name" type="text" />
         <InputWithFormik label="Surname" name="surname" type="text" />
         <div className="sex-wrapper">
-          <SelectWithFormik label="Sex" name="sex" options={sex} />
+          <SelectWithFormik label="Sex" name="sex" options={Object.keys(sex)} />
         </div>
-        <InputWithFormik label="Birthday" name="birthday" type="date" maxAge={new Date().toISOString().substring(0, 10)} />
+        <InputWithFormik
+          label="Birthday"
+          name="birthday"
+          type="date"
+          maxAge={new Date().toISOString().substring(0, 10)}
```

Comment:
Please use DayJs for this
Rename maxAge property as the value here is a current date string...